device/config/functions.go

```diff
@@ -0,0 +1,227 @@
+package config
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/xshellinc/tools/constants"
+	"github.com/xshellinc/tools/dialogs"
+	"github.com/xshellinc/tools/lib/help"
+	"github.com/xshellinc/tools/lib/ssh_helper"
+)
+
+// ConfigLocale is a default method to with dialog to configure the locale
+func ConfigLocale(storage map[string]interface{}) error {
```

Comment:
__[golint]__
* [golint] func name will be used as config.ConfigLocale by other packages, and that stutters; consider calling this Locale

<!-- Comment by [SideCI](https://sideci.com) -->